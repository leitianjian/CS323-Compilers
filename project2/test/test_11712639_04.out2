Error type 4 at Line 7: compare function is redefined
Error type 8 at Line 9: function return type is not matched
Error type 8 at Line 11: function return type is not matched
