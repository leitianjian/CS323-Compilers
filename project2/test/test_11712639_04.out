Error type 4 at line 7: compare function is redefined
Error type 8 at line 9: the function’s return value type mismatches the declared type
Error type 8 at line 11: the function’s return value type mismatches the declared type
